src/metadata/use-option-sets.js

```diff
@@ -0,0 +1,103 @@
+import { useQueryClient, useQuery } from 'react-query'
+import keys from './option-set-query-key-factory.js'
```

Comment:
Just a suggestion: I think the identifier `keys` is a bit ambiguous and when reading this module without having a complete understanding of the code already, the term `keys` take a small amount of time to be interpreted (e.g. "key for what?"; after some reading: "ah.. key for react-query"). Maybe `queryKeys` is better suited, or after renaming the file above this one to `query-key-factory`, this would turn out as: `optionSetQueryKeyFactory.keys` (or so..) which make this very explicit.

Reply:
Yeah I actually saw this when going over the PR as well, and agree. Will rename to `queryKeys`